.github/workflows/release-sdk.yml

```diff
@@ -0,0 +1,33 @@
+name: Release SDK
+
+on:
+  push:
+    tags:
+      - 'v*'
+  workflow_dispatch:
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      id-token: write
+    steps:
+      - uses: actions/checkout@v4
+      - uses: pnpm/action-setup@v4
+        with:
+          run_install: false
+          package_json_file: ./sdk/typescript/package.json
```

Comment:
Why do we need this? Maybe we should just set up the work dir.

Reply:
Wouldn't cache failed to work?